### DISCUSSION TITLE
I can't integrate Supabase because vercel token

### DISCUSSION THREAD

**Discussion by ousszizou** [vercel/vercel#7025] [Help]
Hey folks,
I don't know why supabase can't fetch my vercel's projects !! I can't figure where is the problem exactly.
Even I tried this template here: https://github.com/vercel/nextjs-subscription-payments & didn't works.

Error message: **Retrieve vercel token failed.**

**Comment by gholevas** [comment]
Same issue. "Retrieve vercel token failed"
